src/shared/components/layouts/styles/side-bar-layout.scss

```diff
@@ -51,7 +51,6 @@
 
   &__sidebar-content {
     &--empty {
-      width: 100px;
       height: 100%;
```

Comment:
I know it's not part of the PR either, but wondering if this should have a min-height to prevent it from jumping once ProtVista is loaded? What do you think

Reply:
done